hel_django_auditlog_extra/README.md

```diff
@@ -0,0 +1,330 @@
+# Django Auditlog Extra
+
+A module that fixes some issues and provides some reusable tools for Django application using `django-auditlog` in the context of **City of Helsinki**, that uses **Django Auditlog with Django Graphene** or Django Rest Framework.
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+- [Context](#context)
+  - [Django Auditlog](#django-auditlog)
+  - [Django Graphene](#django-graphene)
+  - [Django Rest Framework](#django-rest-framework)
+- [FAQ](#faq)
+- [Installation](#installation)
+- [Features](#features)
+  - [Context manager](#context-manager)
+    - [`set_request_path`](#set_request_path)
+  - [Middleware](#middleware)
+    - [`AuditlogMiddleware`](#auditlogmiddleware)
+  - [Graphene Decorators](#graphene-decorators)
+    - [`auditlog_access`](#auditlog_access)
+  - [Mixins](#mixins)
+    - [`AuditlogAdminViewAccessLogMixin`](#auditlogadminviewaccesslogmixin)
+  - [Utilities](#utilities)
+    - [`AuditLogConfigurationHelper`](#auditlogconfigurationhelper)
+      - [Initialization examples for `AuditLogConfigurationHelper`:](#initialization-examples-for-auditlogconfigurationhelper)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Context
+
+### Django Auditlog
+
+> PyPi: https://pypi.org/project/django-auditlog/.
+>
+> Github: https://github.com/jazzband/django-auditlog.
+>
+> Docs: https://django-auditlog.readthedocs.io/en/latest/index.html.
+
+The `Django-auditlog` is a reusable app for Django that makes logging changes to your models a breeze. It provides a simple and efficient way to track who made changes to your data and when. This is crucial for accountability, debugging, and compliance with regulations like GDPR.
+
+> NOTE: The `django-auditlog` is supported by the **Platta structured log transfer utility** (https://github.com/City-of-Helsinki/structured-log-transfer).
+
+Here's a breakdown of what it offers:
+
+- **Automatic Change Logging**: It automatically logs changes to your Django models, including creation, updates, and deletion. You can easily track who made the changes and what those changes were.
+
+- **Customization**: You can customize which fields to track, the logging level (e.g., only log changes to specific fields), and even use signals to trigger actions based on logged events.
```

Comment:
> You can customize which fields to track, the logging level (e.g., only log changes to specific fields)

Redundancy as the field selection is twice here, or did you mean something else?